src/plugins/event-plugins/ResourcePlugin.ts

```diff
@@ -72,6 +54,7 @@ export class ResourcePlugin extends InternalPlugin {
         const resourceObserver = new PerformanceObserver((list) => {
             list.getEntries()
                 .filter((e) => e.entryType === RESOURCE)
+                .filter((e) => !this.config.ignore(e))
```

Comment:
question: is the stream...filter pattern for readability?

Reply:
From my perspective, yes, this is about improving readability.